client/python/openlineage/client/run.py

```diff
@@ -34,7 +33,7 @@ class Dataset(RedactMixin):
     name: str = attr.ib()
     facets: Dict[Any, Any] = attr.ib(factory=dict)
 
-    _skip_redact: List[str] = ["namespace", "name"]
+    _skip_redact: ClassVar[List[str]] = ["namespace", "name"]
```

Comment:
Nice to automatically detect and hint this.

Reply:
Not sure if I understand this comment properly.
Are you highlighting it as nice feature or some additional action to do?

Reply:
https://github.com/OpenLineage/OpenLineage/pull/2260/commits/9efdc644f1c1b3dfd2308643428fe5eedc6312ac
seems to do the trick

Reply:
Yes, I mean that's a nice feature 🙂